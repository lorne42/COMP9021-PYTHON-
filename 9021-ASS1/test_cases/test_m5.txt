Sir John said: "Sir Coral or Sir Sandy is a Knave!"
Sir Johnny said: "Sir Johnson, I, Sir August or Sir Jackson is a Knight."
"I am a Knight," whispered Sir Sandy.
Sir Sandy claims: "at most one of Sir John and I is a Knave."
Sir Johnson said: "exactly one of Sir Siren, Sir August and Sir Coral is a Knight."
"At least one of Sir Jackson and I is a Knave," whispered Sir August.
Sir Coral said: "Sir Johnny or Sir Siren is a Knight!"
Sir Jackson shouts: "exactly one of us is a Knight."
Sir Siren said: "Sir Johnson and Sir Johnny are Knights."
