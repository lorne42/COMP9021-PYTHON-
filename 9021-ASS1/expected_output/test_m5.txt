The Sirs are: August Coral Jackson John Johnny Johnson Sandy Siren
There is a unique solution:
Sir August is a Knight.
Sir Coral is a Knight.
Sir Jackson is a Knave.
Sir John is a Knave.
Sir Johnny is a Knight.
Sir Johnson is a Knave.
Sir Sandy is a Knight.
Sir Siren is a Knave.